spec/factories/tasks.rb

```diff
@@ -6,4 +6,16 @@
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
   end
+
+  trait :task_to_be_deleted do
+    title { 'Task to be deleted' }
+  end
+
+  trait :done do
+    status { 'done' }
+  end
+
+  trait :completed_yesterday do
+    completion_date { Time.current.yesterday }
+  end
```

Comment:
この辺はまとめてしまって良いと思います！

ちなみに実装側の仕様でステータスを`done`に変更すると`completion_date`に日付が登録されるので、これらの値の更新はセットで行うようにしましょう。
```suggestion
  trait :done do
    status { :done }
    completion_date { Time.current.yesterday }
  end
```

Reply:
https://github.com/daigomiyoshi/rspec_app_exam/pull/1/commits/5a50026ea01559b2ca7c34586209d02ad47d1c4e にて、`trait`定義をまとめました！